DEVELOPMENT.md

```diff
@@ -73,7 +121,8 @@ style.
 `bootstrap-orbit.{sh,ps1}` performs all the tasks which have to be done once per developer machine.
 This includes:
 * Installing system dependencies
-* Installing conan if necessary.
+* Installing the correct version of conan if necessary.
+* Installing the conan configuration (which changes rarely)
```

Comment:
Add "." at the end for consistency with previous bullet point?

Reply:
done.